app/Http/Controllers/Api/RequestApiController.php

```diff
@@ -42,23 +44,31 @@ public function showSingleRequest(int $requestId): JsonResponse
      * Create a new request for the endpoint.
      * Dispatch ExecuteRequestJob.
      *
-     * @param Request $request
+     * @param WebRequest $webRequest
```

Comment:
I don't like this rename... but I also don't have a better one. It's good for now. 😬